App/frontend/src/components/Homepage/Cards.tsx

```diff
@@ -120,14 +120,14 @@ export const TextFieldCard = ({ className, ...props }: CardProps) => {
           textAlign: "start",
         }}
       >
-        <Title3 
+       <h2 
           style={{
             fontSize: "1.2rem",
-            marginBottom: "2px",
+            fontWeight: "500"
```

Comment:
@Rohini-Microsoft - Please check the impacted places as mentioned in the comments

Reply:
Resolved